### ISSUE TITLE
About the user mannual

### ISSUE THREAD

**Issue by sunpengmri** [kmjohnson3/4D-Flow-Tools#3]
Dear Johnson,

Thanks a lot for sharing this wonderful code. Could you kindly share a user mannual with me?

Best,
Peng

**Comment by kmjohnson3** [comment]
I'm not sure we have a great manual but we might have videos documenting processing steps. Was there something in particular that you wanted to use this software for?

**Comment by sunpengmri** [comment]
> I'm not sure we have a great manual but we might have videos documenting processing steps. Was there something in particular that you wanted to use this software for?

Yes, I would  very appreciate if you could share some instructions about how to run this software. We are doing some research on 4d flow based CFD, so your great code would be  a very good start. Thanks a lot again!
